vagrant.rb

```diff
@@ -130,18 +130,18 @@ def configure(config, os, pf_name, pf_id, host_name, host_id,
     command = "/vagrant/scripts/cleanbox.sh\n"
     command += "/vagrant/scripts/network.sh #{net} \"@host_list@\"\n"
     if setup != "empty" and setup != "ncf" then
-      command += "ALLOWEDNETWORK=#{net}.0/24 /usr/local/bin/rudder-setup setup-#{setup} \"#{version}\" \"#{server}\"\n"
```

Comment:
The grep remove lines with single dots, but also wait for a full line of input before displaying output.
This avoid vagrant displaying a new line for each dot